packages/paste-core/components/combobox/src/styles/ComboboxListboxOption.tsx

```diff
@@ -1,74 +1,107 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
 import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {SelectedIcon} from '@twilio-paste/icons/esm/SelectedIcon';
 import type {BoxProps, BoxStyleProps} from '@twilio-paste/box';
 import {Text} from '@twilio-paste/text';
 import type {PositionOptions} from '@twilio-paste/style-props';
-import type {VirtualItem} from 'react-virtual/types';
+import type {VirtualItem} from 'react-virtual';
 
 export interface ComboboxListboxOptionProps extends Pick<BoxProps, 'element'> {
   children: NonNullable<React.ReactNode>;
   highlighted?: boolean;
+  selected?: boolean;
+  disabled?: boolean;
   variant: 'default' | 'groupOption';
   startHeight?: VirtualItem['start'];
 }
 
 const VariantStyles: {[key in ComboboxListboxOptionProps['variant']]: BoxStyleProps} = {
   groupOption: {
-    paddingLeft: 'space90',
-    paddingRight: 'space90',
+    paddingLeft: 'space70',
+    paddingRight: 'space50',
   },
   default: {
-    paddingLeft: 'space70',
-    paddingRight: 'space70',
+    paddingLeft: 'space50',
+    paddingRight: 'space50',
   },
 };
 
+const getVirtualStyles = (startHeight: number): Record<string, unknown> => ({
+  position: 'absolute' as PositionOptions,
+  top: 0,
+  left: 0,
+  width: '100%',
+  transform: `translateY(${startHeight}px)`,
+});
+
 const ComboboxListboxOption = React.forwardRef<HTMLLIElement, ComboboxListboxOptionProps>(
-  ({children, element = 'COMBOBOX', highlighted, variant = 'default', startHeight, ...props}, ref) => {
-    const virtualItemStyles =
-      startHeight !== undefined
-        ? {
-            position: 'absolute' as PositionOptions,
-            top: 0,
-            left: 0,
-            width: '100%',
-            transform: `translateY(${startHeight}px)`,
-          }
-        : {};
+  (
+    {
+      children,
+      element = 'COMBOBOX',
+      highlighted,
+      selected,
+      disabled = false,
+      variant = 'default',
+      startHeight,
+      ...props
+    },
+    ref
+  ) => {
+    const virtualItemStyles = startHeight != null ? getVirtualStyles(startHeight) : {};
+
     return (
       <Box
         {...safelySpreadBoxProps(props)}
+        // FIX: Manually control aria-selected state to be true when an item has been selected.
+        // Downshift sets aria-selected to true when an item is highlighted, which we don't want.
+        aria-selected={selected}
```

Comment:
issue: weirdly this is getting rendered as `[Object object]`

![image](https://user-images.githubusercontent.com/368249/191147245-d98911cd-6228-4212-89ec-2b5dc9ab1fd4.png)

Reply:
Good catch, fixed